crates/routes/src/images.rs

```diff
@@ -225,9 +226,22 @@ pub async fn image_proxy(
   // for arbitrary purposes.
   RemoteImage::validate(&mut context.pool(), url.clone().into()).await?;
 
+  if context
+    .settings()
+    .pictrs_config()?
+    .proxy_bypass_domains
+    .iter()
+    .any(|s| url.domain().expect("Invalid URL").starts_with(s))
+  {
+    return Ok(
```

Comment:
Can just do `Ok(....)`

Reply:
`return` is required here because the if statement is not the last expression